docs/01-installing.md

```diff
@@ -18,25 +24,47 @@ All the new features with all the new bugs:
 LVBRANCH=rolling bash <(curl -s https://raw.githubusercontent.com/lunarvim/lunarvim/rolling/utils/installer/install.sh)
 ```
 
-Make sure you have the newest version of Neovim (0.5).
-
 ```bash
 bash <(curl -s https://raw.githubusercontent.com/LunarVim/LunarVim/rolling/utils/installer/install-neovim-from-release)
 ```
 
-After installation run `lvim` and then `:PackerInstall`
+## After installation
```

Comment:
There's already a section for `after-install` procedures.  This doesn't need to be here.